### PR TITLE
Brand name update to Wise

### PR DESCRIPTION
**Pull request by adsingh14**
Official link: https://wise.com/gb/blog/world-meet-wise

**Issue:** #7002 
**Alexa rank:** 477
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

~~Rename file/icon name.~~ Updated icon 

wise

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
Hi @adsingh14,

Can you check the logo again? I used the officel svg `https://wise.com/public-resources/assets/logos/wise/brand_logo_inverse.svg`

image

**Comment by adamrusted** [comment]
We also won't be able to merge this until a major release in it's current state. Could you please split this into _two_ PRs? One with the new Wise icon, and another removing TransferWise? That way we can implement the new icon sooner, and deprecate the old icon from v7.

**Comment by adsingh14** [comment]
> Hi @adsingh14,
> 
> Can you check the logo again? I used the officel svg `https://wise.com/public-resources/assets/logos/wise/brand_logo_inverse.svg`
> 
> image
> 

Okay, I'll create a new branch.

**Comment by adsingh14** [comment]
> We also won't be able to merge this until a major release in it's current state. Could you please split this into _two_ PRs? One with the new Wise icon, and another removing TransferWise? That way we can implement the new icon sooner, and deprecate the old icon from v7.

Okay, I'll create a new PR with current name.

**Comment by adamrusted** [comment]
@adsingh14 we can keep this open, but repurpose the PR for the removal of TransferWise. That way, we'll remember it needs doing when we are working on the next big release!

**Comment by adamrusted** [comment]
If you restore the branch, we can reopen the PR and use the base from this to implement the removal. Do you want to do that, or shall I open a new PR for its removal?

File: icons/wise.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Wise</title><path d="m3.6426 0 3.7383 6.3594-6.6602 6.3613H12.043l1.1816-2.7734H7.4883l3.5879-3.588-2.084-3.5878h9.7324L9.7441 24h3.373L23.2794 0Z"/></svg>
```
Comment (adamrusted) [inline icons/wise.svg:1]:
```suggestion
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -11058,6 +11053,12 @@
             "hex": "1679A7",
             "source": "https://gitlab.com/wanduow/wireshark/-/blob/cd5539b0f76975474869984a9d2f0fce29d5c21e/image/wsicon.svg"
         },
+        {
+            "title": "Wise",
+            "hex": "00B9FF",
+            "source": "https://wise.com",
+            "guidelines": "https://brand.wise.com/logo"
+        },
```
Comment (adamrusted) [inline _data/simple-icons.json:11061]:
```suggestion
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
@adsingh14 - I've just highlighted the two segments of code that'd need to go in order for this to be a 'removal' PR. Once that's done, I'll tag this as a v7 PR and we can leave it until the next release!

**Comment by adsingh14** [comment]
> @adsingh14 - I've just highlighted the two segments of code that'd need to go in order for this to be a 'removal' PR. Once that's done, I'll tag this as a v7 PR and we can leave it until the next release!

I think this PR is unnecessary because of recent 'Wise' PR ! Removing this branch would automatically target to the current 'TransferWise' icon. Therefore, I'm removing this branch.

**Pull request closed without merging**